core/src/main/java/ai/timefold/solver/core/impl/constructionheuristic/DefaultConstructionHeuristicPhase.java

```diff
@@ -165,19 +175,24 @@ public void solvingError(SolverScope<Solution_> solverScope, Exception exception
         decider.solvingError(solverScope, exception);
     }
 
-    public static class Builder<Solution_> extends AbstractPhase.Builder<Solution_> {
+    public static class DefaultConstructionHeuristicPhaseBuilder<Solution_>
```

Comment:
The name `Builder` is used by other builders, e.g., `DefaultLocalSearchPhase`. Is there a reason for changing this one?

Reply:
This is exactly the reason to change the name. :-) When looking for the builder in IntelliJ, I type "Builder" and get several different classes with exactly the same name. This is not practical.